rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -0,0 +1,228 @@
+            description: "Create a forum post with the following specifications:
+
+1. Title Format: 
+   'NNS Updates <ISO 8601 DATE>(: <Anything interesting to announce>)'
+
+2. Category: 
+   Governance > NNS proposal discussion
+   Reference: https://forum.dfinity.org/t/nns-proposal-discussions/34492
+
+3. Tags:
+   - Protocol-canister-management / Service-nervous-system-management
+   - nns / sns
+
+4. Content:
+   - Link to proposals in IC Dashboard
+   - Include all proposal texts
+   - Use six consecutive backticks (```````) to wrap proposal text
+   - Call out any 'interesting' changes, breaking changes, or required actions
+
+5. Generate Forum Content:
+   If your proposals are in a dedicated directory:
+
+   For NNS upgrades:
+   ```bash
+   ./testnet/tools/nns-tools/cmd.sh \\
+       generate_forum_post_nns_upgrades \\
+       $PROPOSALS_DIR/nns-*.md \\
+       | pbcopy
+   ```
+
+   For SNS WASM publishing:
+   ```bash
+   ./testnet/tools/nns-tools/cmd.sh \\
+       generate_forum_post_sns_wasm_publish \\
+       $PROPOSALS_DIR/sns-*.md \\
+       | pbcopy
+   ```
+
+6. Required Follow-ups:
+   - Reply to NNS Updates Aggregation Thread (https://forum.dfinity.org/t/nns-updates-aggregation-thread/23551)
+   - If SNS canister WASMs were published, update SNS Upgrades Aggregation Thread
+     (https://forum.dfinity.org/t/sns-upgrade-aggregation-thread/24259/2)",
+        },
+        Step {
+            title: "Schedule Trusted Neurons Vote",
+            description: "Schedule calendar event for Trusted Neurons to vote the following Monday.
+
+Calendar Event Setup:
+1. Duplicate a past event from:
+   https://calendar.google.com/calendar/u/0/r/eventedit/duplicate/MjJvMTdva2xtdGJuZDhoYjRjN2poZzNwM2ogY182NGYwZDdmZDYzYjNlMDYxZjE1Zjk2MTU1NWYzMmFiN2EyZmY3M2NjMWJmM2Q3ZTRkNGI3NGVjYjk1ZWVhM2M0QGc
+
+2. Use 'NNS Upgrades' calendar
+
+3. Timing:
+   - Usually scheduled at 6 pm Central European Time
+   - For multiple proposals, schedule separate sequential events
+
+4. Required Fields:
+   - Date and Time
+   - Title: Include canister name and proposal ID
+   - Description: Link to the proposal
+
+5. Actions:
+   - Click 'Save' to create event
+   - Send email invitations when prompted
+   - If people don't respond, ping @trusted-neurons in #eng-release channel",
+        },
+        Step {
+            title: "Post to Slack",
+            description: "1. Post to #eng-nns-releases:
+   - Paste the proposal text
+   - Include dashboard link (https://dashboard.internetcomputer.org/proposal/<PROPOSAL_ID>)
+   - Command to copy proposal: cat /path/to/proposal_text.md | pbcopy
+
+2. For bigger feature releases:
+   - Ping comms team in #social-media
+   - They may share additional information over forum/socials
+   - They can encourage manual voting",
+        },
+        Step {
+            title: "Alert FITS",
+            description: "Important: Upgrades might trigger IC_NNS_SubnetMessageInstructionsNearLimit or similar alerts
```

Comment:
Have we been observing this?  I am not aware that this has been part of our regular weekly practice

Reply:
removed!